class.jetpack-xmlrpc-server.php

```diff
@@ -70,9 +70,31 @@ function xmlrpc_methods( $core_methods ) {
 	function bootstrap_xmlrpc_methods() {
 		return array(
 			'jetpack.verifyRegistration' => array( $this, 'verify_registration' ),
+			'jetpack.remoteAuthorize' => array( $this, 'remote_authorize' ),
 		);
 	}
 
+	function remote_authorize( $request ) {
```

Comment:
We need something to tie the request to the user, then. The nonce did that previously, though we were all wary of relying on it exclusively.